Running-Mastodon/Production-guide.md

```diff
@@ -37,7 +37,9 @@ server {
   server_name example.com;
 
   ssl_protocols TLSv1.2;
-  ssl_ciphers HIGH:!MEDIUM:!LOW:!aNULL:!NULL:!SHA;
+  ssl_ciphers 'ECDHE+CHACHA20:ECDHE+AESGCM'; # or 'ECDHE:!DSS:!PSK:!SRP:!3DES:!RC4:!DES:!IDEA:!RC2:!NULL'
```

Comment:
This is what `EECDH+AES`. Sort with GCM / CCM / CBC, then AES-256 / AES-128, then SHA-384 / SHA-256 / SHA-1. No need to specify explicit AESGCM priority.